docs/source/installation.rst

```diff
@@ -38,15 +117,15 @@ RADICAL-Pilot is distributed via PyPi and Conda-Forge. To install RADICAL-Pilot
 to a virtual environment do:
 
 via PyPi
------------------
+--------
 .. code-block:: bash
 
     virtualenv --system-site-packages $HOME/ve
     source $HOME/ve/bin/activate
     pip install radical.pilot
 
 via Conda-Forge
------------------
+---------------
 .. code-block:: bash
 
     conda create -n ve -y python=2.7
```

Comment:
I think the `source activate ve` line changed, but @iparask should confirm / refute that.

Reply:
@iparask confirmed that is the right line.

Reply:
Thanks for checking!